storage/store.go

```diff
@@ -0,0 +1,28 @@
+package storage
+
+type KVStore interface {
+    // key value
+    Put(key string, value string) error
+    Get(key string) (string, error)
+    Del(key string) error
+    Contains(key string) bool
+}
+
+type QueueStore interface {
+    // append new object
+    Enqueue(key string, value string) error
+    Peek() (string, error)
+    Dequeue() (string, error)
+
```

Comment:
Regarding the queue store, the actual usage pattern is not like using a queue. What we need are only:

- put an id in
- get a list of limited number of ids which is relatively older
- delete an id

So, using queue pattern looks not very matching.